src/auth/shibboleth.js

```diff
@@ -3,15 +3,14 @@ const { createOrUpdateUser, addJwtCookie, isSafeUrl } = require('./util')
 const safeAsync = require('../middleware/safeAsync')
 
 module.exports = safeAsync(async (req, res) => {
-  // Get the user's NetID based on the "eppn" header
-  const email = req.get('eppn') || ''
-  if (email.indexOf('@') === -1) {
+  // Get the user's email based on the "eppn" header
+  const uid = req.get('eppn') || ''
+  if (!uid.endsWith('@illinois.edu')) {
```

Comment:
Do you really need to check for @illinois.edu, or can you just check for any email address? If you feel there is a need to check the domain, can it be parameterized?

Reply:
@nwalters512

Reply:
It looks like our [Shib documentation](https://answers.illinois.edu/page.php?id=48435&no_frill=1) asks to make sure that the eppn attribute ends with @illinois.edu, so I've just exposed it as an environment variable (`EPPN_SUFFIX`).